app/grandchallenge/algorithms/templates/algorithms/job_list_row.html

```diff
@@ -66,11 +67,15 @@
 <split></split>
 
 {% if object.status == object.SUCCESS %}
-    <a class="badge badge-primary"
-       title="View algorithm result"
-       href="{% url 'workstations:workstation-session-create' slug=algorithm.workstation.slug %}?{% workstation_query algorithm_job=object config=algorithm.workstation_config %}">
+    <button class="btn badge badge-primary"
+            title="View algorithm result"
+            data-session-control
+            data-session-origin="https://us-east-1.{{ request.site.domain }}"
+            data-create-session-url='{% url 'workstations:workstation-session-create' slug=object.algorithm_image.algorithm.workstation.slug %}'
+            data-workstation-query='{% workstation_query algorithm_job=object config=object.algorithm_image.algorithm.workstation_config %}'
+            data-workstation-window-identifier='workstation-{{ object|meta_attr:'app_label' }}-{{ object|meta_attr:'model_name' }}'>
```

Comment:
Is model_name necessary here?

Reply:
From our discussions with Chris yesterday, I would we say we don't need the model name. All we want to be able to distinguish is reader_study sessions from algorithm from archive sessions. The app_label will do for that.